js/messaging.js

```diff
@@ -307,14 +297,13 @@ function isTabReplaceable(tab, newUrl) {
   return true;
 }
 
-function activateTab(tab, {url, index} = {}) {
+function activateTab(tab, {url, index, openerTabId} = {}) {
   const options = {active: true};
-  if (url) {
-    options.url = url;
-  }
+  if (url) options.url = url;
+  if (openerTabId != null) options.openerTabId = openerTabId;
```

Comment:
I think we also have to check `openerTabIdSupport` here?

Reply:
Yeah, I lost it along the way.